### PR TITLE
NN-1615 appointment form tweaks

### PR DESCRIPTION
**Pull request by JamieDraperUK**

### CODE REVIEW

File: src/BulkAppointments/AppointmentForm.js [hunk 1]
```diff
@@ -34,32 +34,32 @@ export const onSubmit = onSucces => values => {
...
+    formErrors.push({ targetName: 'location', text: 'Select location' })
   }
 
   if (!values.date) {
     formErrors.push({ targetName: 'date', text: 'Date is required' })
   }
 
   if (isToday && moment(values.startTime).isBefore(now)) {
-    formErrors.push({ targetName: 'startTime', text: "Start time shouldn't be in the past" })
+    formErrors.push({ targetName: 'startTime', text: 'Start time must be in the future' })
```
Comment (petergphillips) [inline src/BulkAppointments/AppointmentForm.js]: this message doesn't correspond to the validation.  Start time can be now can't it?

Comment (JamieDraperUK) [reply src/BulkAppointments/AppointmentForm.js]: Good point. It was better before. I'll change it back.

Comment (petergphillips) [reply src/BulkAppointments/AppointmentForm.js]: although you can't use the word `shouldn't` as it implies a choice, `must` is better

File: src/BulkAppointments/AppointmentForm.js [hunk 2]
```diff
@@ -34,32 +34,32 @@ export const onSubmit = onSucces => values => {
   }
 
   if (!values.appointmentType) {
-    formErrors.push({ targetName: 'appointmentType', text: 'Appointment type is required' })
+    formErrors.push({ targetName: 'appointmentType', text: 'Select appointment type' })
   }
 
   if (!values.location) {
-    formErrors.push({ targetName: 'location', text: 'Location is required' })
+    formErrors.push({ targetName: 'location', text: 'Select location' })
```
Comment (petergphillips) [inline src/BulkAppointments/AppointmentForm.js]: I think `Select a location` would fit more with the GDS style messages, they have things like `Enter a postcode`

File: src/BulkAppointments/AppointmentForm.js [hunk 3]
```diff
@@ -34,32 +34,32 @@ export const onSubmit = onSucces => values => {
...
   }
 
   const DATE_TIME_FORMAT_SPEC = 'YYYY-MM-DDTHH:mm:ss'
 
   if (
     moment(values.endTime, DATE_TIME_FORMAT_SPEC).isBefore(moment(values.startTime, DATE_TIME_FORMAT_SPEC), 'minute')
   ) {
-    formErrors.push({ targetName: 'startTime', text: "Start time should't be after End time" })
-    formErrors.push({ targetName: 'endTime', text: "End time shouldn't be before Start time" })
+    formErrors.push({ targetName: 'startTime', text: 'The start time must be before the end time' })
```
Comment (petergphillips) [inline src/BulkAppointments/AppointmentForm.js]: you haven't got `The` at the start of any of the other messages, so not needed here.
It is also odd that you are indicating that both fields are in error, whereas it is likely that just the end time is at fault.  I would have just highlighted the end time.

File: src/BulkAppointments/AppointmentForm.js [hunk 4]
```diff
@@ -34,32 +34,32 @@ export const onSubmit = onSucces => values => {
...
+    formErrors.push({ targetName: 'endTime', text: 'The end time must be after the start time' })
```
Comment (petergphillips) [inline src/BulkAppointments/AppointmentForm.js]: Can the end time be the same as the start time?  Validation allows it

File: src/BulkAppointments/AppointmentForm.js (hunk 2 above)
Comment (JamieDraperUK) [reply src/BulkAppointments/AppointmentForm.js]: The wording is from Clare. I'll catch up with her to see what she thinks

File: src/BulkAppointments/AppointmentForm.js (hunk 3 above)
Comment (JamieDraperUK) [reply src/BulkAppointments/AppointmentForm.js]: The following text has been supplied by Clare.
'The start time must be before the end time’
’The end time must be after the start time

Comment (petergphillips) [reply src/BulkAppointments/AppointmentForm.js]: You don't need both conditions if you reverse the logic so:
```
const isNotAfter = ! moment(values.endTime, DATE_TIME_FORMAT_SPEC).isAfter(
    moment(values.startTime, DATE_TIME_FORMAT_SPEC),
    'minute'
  )
```
although then it becomes clear that the variable should be called `endNotLaterThanStart` or similar

**Pull request merged**